ddtrace/internal/uds.py

```diff
@@ -4,17 +4,19 @@
 from ..compat import httplib
 
 
-class UDSHTTPConnection(httplib.HTTPConnection):  # type: ignore[name-defined]
+class UDSHTTPConnection(httplib.HTTPConnection):
     """An HTTP connection established over a Unix Domain Socket."""
 
     # It's "important" to keep the hostname and port arguments here; while there are not used by the connection
     # mechanism, they are actually used as HTTP headers such as `Host`.
-    def __init__(self, path, https, *args, **kwargs):
-        # type: (str, bool, *Any, **Any) -> None
-        if https:
```

Comment:
idek if it's possible to have an https connection over unix socket or if it's even something we want to support (we definitely don't test it) but should this change be made subtly in this PR?

Reply:
In theory, I think it's possible.
In practice we don't support it since we don't support something like `unix+ssl://` as a scheme.